01/TableBody.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { v4 as uuid } from "uuid";
+import TableRow from "./TableRow";
+
+class TableBody extends React.Component {
+    render() {
+        const { data } = this.props;
+
+        return data.map((p) => (
+            <TableRow
+                key={uuid()}
```

Comment:
Lepiej `uuid()` dodać do danych (czyli tutaj produktów) i je wykorzystywać z `props`.
Obecnie za każdym razem (po ponownym przerenderowaniu) `key` jest inny więc nie wiele nam to daje.